### ISSUE TITLE
error with resize and dither (error diffusion)

### ISSUE THREAD

**Issue by simolg** [chaiNNer-org/chaiNNer#1999]
- Chainner version: 0.18.9
- OS: Windows 10

**Description**

this is what i try to do : Load image -> resize  -> color reduction (with dither, whatever error diffusion maps)
this error occurs:
```
Error
An error occurred in a Dither (Palette) node:
assignment destination is read-only

Input values (partial):
- Image: RGB image 320x180
- Palette: RGB image 8x1
- History Length: 8
```

important note:
if i load the image and directly dither (_without the resize_) everything works

![Capture_02](https://github.com/chaiNNer-org/chaiNNer/assets/103586111/407adf77-5f6b-4846-9cd6-8a6cdba3b66b)

**Logs**
[logs.zip](https://github.com/chaiNNer-org/chaiNNer/files/12188215/logs.zip)

**Comment by joeyballentine** [comment]
This is definitely a bug. Thanks for reporting it!

**Comment by RunDevelopment** [comment]
This has been fixed by #1844, and will be part of the next release.
